README.md

```diff
@@ -68,15 +79,15 @@ Also if you're having issues with it not detecting the shims you've installed it
 
 ## USAGE
 
-These instructions are for a specific version of asdf. Make sure the version README you refer to matches the version of asdf you have installed. Checkout a specific tagged version in the GitHub branches/tags drop-down if you need to.
+These instructions are for a specific version of asdf. Make sure the version
+README you refer to matches the version of asdf you have installed. Checkout a
+specific tagged version in the GitHub branches/tags drop-down if you need to.
 
 ### Manage plugins
 
-Plugins are how asdf understands how to handle different packages.
-
-You can find a list of all asdf plugins in the [plugins repository](https://github.com/asdf-vm/asdf-plugins).
-
-There is a [super-simple API](https://github.com/asdf-vm/asdf/blob/master/docs/creating-plugins.md) for supporting more languages.
+Plugins are how asdf understands how to handle different packages. The [plugins
```

Comment:
_This didn't seem like it should be 3 separate paragraphs, so I reworded it a bit and joined it into one._